specs/protocol/configurability.md

```diff
@@ -0,0 +1,76 @@
+# OP Stack Configurability
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Consensus Parameters](#consensus-parameters)
+- [Admin Roles](#admin-roles)
+- [Service Roles](#service-roles)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+When deploying the OP Stack software to a production environment,
+certain parameters about the protocol can be configured. These
+configurations can include a variety of parameters which affect the
+resulting properties of the blockspace in question. 
+
+There are four categories of OP Stack configuration options:
+
+- **Consensus Parameters**: Parameters and properties of the chain that may
```

Comment:
With this definition, it sounds some of the line items in the `Consensus Parameters` table below are out of place now. 

Anything that can be upgraded with admin keys should fall under policy right?

Reply:
The structure of Ben's doc is: https://www.notion.so/Superchain-Segmentation-33a3152ca1dd4e87a40af52e08aae75d

<img width="451" alt="Screenshot 2024-03-20 at 17 42 47" src="https://github.com/ethereum-optimism/specs/assets/15608778/ce949dae-170c-4a82-ba82-d9e9f074af63">

Reply:
> Anything that can be upgraded with admin keys should fall under policy right?

Not necessarily - consensus enforces the particular batcher address via the "batcher hash" config. The identity that owns the system config isn't related to consensus, it can be a smart contract or eoa altho we recommend a safe smart contract wallet